docs/modules/ROOT/pages/gsn.adoc

```diff
@@ -0,0 +1,93 @@
+= The Gas Station Network
+
+User onboarding is one of the hottest topics in Ethereum, and meta-transactions the de-facto solution to this important problem. Here you'll learn all about the Gas Station Network (GSN) and how to write contracts that don't require their users to hold Ether to pay for gas.
```

Comment:
Should "and meta-transactions the de-facto solution"
be "and meta-transactions is the de-facto solution"?